### PR TITLE
Date picker added

### PR DESCRIPTION
**Pull request by pdolega**
http://mestudent.github.io/react-formalin/sample/page/sample.html#date

### CODE REVIEW

File: sample/SamplePage.tsx [hunk 1]
```diff
@@ -71,10 +76,8 @@ class SamplePage extends React.Component<void, State> {
...
 
-window["initSamplePage"] = function() {
-    console.debug(`Sample page initialization`);
-    ReactDOM.render(
-        <SamplePage/>,
-        document.getElementById('react')
-    );
-};
\ No newline at end of file
+console.debug(`Sample page initialization`);
```
Comment (pdolega) [inline sample/SamplePage.tsx]: no more jquery needed to mount top react element

File: sample/page/sample.html [hunk 2]
```diff
@@ -7,13 +7,6 @@
         <div id="react"></div>
     </body>
 
-    <script src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
```
Comment (pdolega) [inline sample/page/sample.html]: yep - no more jquery

File: webpack.config.js [hunk 3]
```diff
@@ -23,7 +23,10 @@ module.exports = [
         },
         resolve: {
             extensions: ["", ".ts", ".tsx", ".js", ".html"],
-            modulesDirectories: ["node_modules"]
+            modulesDirectories: ["node_modules"],
+            alias: {
```
Comment (pdolega) [inline webpack.config.js]: jquery needs to be visible globally for bootstrap-datepicker

**Comment by gkurdziel** [comment]
:+1:

**Pull request merged**